apps/i18n/common/en_us.json

```diff
@@ -1488,6 +1488,10 @@
   "incubator_codebridge_python_desc": "Try out Code.org's first Python Code Editor.",
   "incubator_codebridge_python_project_button": "Create Your Own Project",
   "incubator_codebridge_python_title": "Codebridge: Python",
+  "incubator_genai_title": "Exploring Generative AI",
+  "incubator_genai_desc": "Try our new course that prepares students to explore and create with generative AI, developing critical thinking and ethical awareness",
```

Comment:
"prepares" sounds like it gets students ready to do the exploration and creation in a separate activity.  Maybe "allows"?

Reply:
Changed to allows